docs/flinkDev/building.zh.md

```diff
@@ -22,62 +22,63 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-This page covers how to build Flink {{ site.version }} from sources.
+本篇主题是如何从版本 {{ site.version }} 的源码构建 Flink。
 
 * This will be replaced by the TOC
 {:toc}
 
-## Build Flink
+## 构建 Flink
 
-In order to build Flink you need the source code. Either [download the source of a release]({{ site.download_url }}) or [clone the git repository]({{ site.github_url }}).
+首先需要准备一份源码。有两种方法：第一种参考 [从发布版本下载源码]({{ site.download_url }})；第二种参考 [从 Git 库克隆 Flink 源码]({{ site.github_url }})。
 
-In addition you need **Maven 3** and a **JDK** (Java Development Kit). Flink requires **at least Java 8** to build.
+还需要准备，**Maven 3** 和 **JDK** (Java开发套件)。Flink至少依赖 **Java 8** 来进行构建。
 
-*NOTE: Maven 3.3.x can build Flink, but will not properly shade away certain dependencies. Maven 3.2.5 creates the libraries properly.
+*注意：Maven 3.3.x 可以构建 Flink，但是不会去掉指定的依赖。Maven 3.2.5 可以很好地构建对应的库文件。
```

Comment:
```suggestion
*注意：Maven 3.3.x 可以构建 Flink，但是不能正确地 shade 指定的依赖。Maven 3.2.5 可以正确地构建库文件。
```

@wuchong how to translate "shade"?

@wbgentleman "properly" here means "正确地" IMO, because actually you cannot build Flink with Maven 3.3.x, which will bring some subtle troubles.

Reply:
as below I think phrase like "依赖屏蔽" should be ok.

Reply:
Can we translate shade to "屏蔽"？ @TisonKun @wuchong 

`@wbgentleman "properly" here means "正确地"` - Accept

Reply:
Can we translate shade to "屏蔽"？ - Good to me.